docs/markdown/foundations/content_standards.md

```diff
@@ -0,0 +1,928 @@
+
+<TwoCol>
+  <Group>
+    <Do title="Do examples" />
+      - UI: You can have up to 6 secret boards
+      - Longform: You can have up to six secret boards
+      - The top 10 tricks to making successful Pins for your business
+  </Group>
+  <Group>
+  <Dont title="Don't examples" />
+     - UI: You can have up to six secret boards
+     - The top ten tricks to making successful Pins for your business
+  </Group>
+</TwoCol>
+---
+
+### Timestamps
+Timestamps should make sense when you scan them—no one should have to do lots of mental math to figure out when something happened. In small spaces (like messages) we can abbreviate. 
+
+<TwoCol>
+  <Group>
+    <Do title="Do" />
+      - Just now! up to 59 seconds
+      - 1 minute/X minutes up to an hour
+      - 1 hour/X hours up to a day 
+      - 1 day/X days up to a week
+      - 1 week/X weeks up to a month 
+      - After a month, switch to a MM/DD/YY date
+  </Group>
+  <Group>
+  <Dont title="Don't" />
+     - 3 seconds
+     - 96 minutes ago
+     - 30 min
+     - 36 weeks ago
+  </Group>
+</TwoCol>
+
+**Using timestamps in small spaces...**
+- Xm
+- Xh
+- Xd
+- Xw
+- After a month, switch to a MM/YY date
+<br/>
+**Using timestamps in Ads Manager (including graphs)...**
+- Use the 24-hour clock structure
+- 00:00 is midnight
+- 23:59 is the last minute of the day
+- Use PST/PDT throughout, not UTC
+
+---
+
+## Grammar and punctuation
+
+### Capitalization
+Sentence case is easy to read, and feels way more conversational (vs. title case, which feels very Grand and Self Important). It’s our go-to approach, with only a few exceptions.
+
+**Use sentence case**
+- Board name and section name placeholder text
+- Business site and blog post headlines
+- Buttons
+- Error message titles
+- Titles of talks at events 
+- Form field titles
+- Links
+- Memo titles
+- Menu items
+- Microcopy and helper text
+- Navigation 
+- Page titles
+- Placeholder text (gray instructional text within form fields)
+- Sub-headlines
+- Tabs/sections of Pinterest (following feed, more ideas tab, etc.)
+<br/>
+**Use title case**
+- Brands: If you must refer to a brand, make sure to capitalize it (“Crock-Pot” rather than “crockpot”) and in the third person use “they” not “it”
+- Holidays (you can always Google to double check tricky ones like Father’s Day or Back to School)
+- Job titles
+- Pin: Always capitalize—it’s a legal thing (see How to use Pin for more!)
+- Pinner (Note: Use this sparingly, and ideally not at all. Instead, say “people” or “people on Pinterest”) 
+- Proper nouns (World War II, Scarlett Johansson), acronyms (NFL, DIY)
+- Topics in product UI/New User Activation (ex: Weddings, DIY, Naruto and Anime)
+- Generation names (ex: Baby Boomers, Gen X)
+
+---
```

Comment:
We don't typically use horizontal lines in our docs pages, I would recommend removing these and just ensuring there's enough white space b/w sections

Reply:
Ah good catch, we can remove them